scripts/test_translator.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, path: str, link_name: str,
 
 
 class CFile:
-    def __init__(self, log_level: str, path: str, flags: Set[str] = None) -> None:
+    def __init__(self, log_level: str, path: str, flags: Set[str] = set()) -> None:
```

Comment:
`set()` is mutable, so this will re-use the same set on separate `__init__` calls.  I think you have to set it to `None` and then do an

```python
if flags is not None:
    flags = set()
```

Very annoying, but oh well.

Reply:
We don't mutate the set, so it's fine.